### ISSUE TITLE
Projects Gone?

### ISSUE THREAD

**Issue by quizque** [xem/3DShomebrew#28]
I was wondering what happened after all I'm sorta a noob in coding home brews and this tutorial helped me a lot. I sorta just stop working on it for a bit(about 6 months) and all progress has came to a halt. It would be very sad to see such a good tutorial to go to wast :-1:  :(

**Comment by xem** [comment]
Yeah, sorry about that. Many people asked the same thing but the truth is, I'm also a noob in 3DS dev. I created this tutorial when ctrulib was created by smealum, and I kinda hoped ctrulib would get documente
d enough to let me continue this tutorial, though it never really happened and it's still too complicated for me to learn it, use it, and teach it. But the wiki is public and anyone can contribute to it.
